PivotalCloudFoundry/Page_Cloud Foundry.json

```diff
@@ -3903,6 +3903,7 @@
   "sf_dashboard" : "Cloud Controller Overview",
   "sf_description" : "Overview of the cloud controller component.",
   "sf_discoveryQuery" : "job:cloud_controller_worker metric_source:cloudfoundry",
+  "sf_discoverySelectors" : [ "_exists_:host" ],
```

Comment:
"_exists_:host" is only used in the UI. For SD purposes, add "sf_key:host" to that list.
This is an unfortunate limitation that needs to be addressed at some point.